packages/starlight/__tests__/remark-rehype/asides.test.ts

```diff
@@ -106,6 +106,35 @@ Nested tip.
 	expect(res.code).toMatchFileSnapshot('./snapshots/nested-asides.html');
 });
 
+test('nested asides with custom titles', async () => {
+	const res = await processor.render(`
+:::::caution[Caution with a custom title]
+Nested caution.
+
+::::note
+Nested note.
+
+:::tip[Tip with a custom title]
+Nested tip.
+:::
+
+::::
+
+:::::
+`);
+	const labels = [...res.code.matchAll(/aria-label="(?<label>[^"]+)"/g)].map(
+		(match) => match.groups?.label
+	);
+	expect(labels).toMatchInlineSnapshot(`
+		[
+		  "Caution with a custom title",
+		  "Note",
+		  "Tip with a custom title",
+		]
+	`);
```

Comment:
Even if the snapshot can be used to ensure custom titles are properly rendered, I added this small expectation to make it easier to glance at what the expected output is instead of having to read the snapshot file.